src/views/DissolutionFirm/DissolutionFirm.vue

```diff
@@ -427,6 +447,18 @@ h2::before {
   background-color: $gray1;
 }
 
+.red-bar-error {
+  border-left: 3px solid $app-red !important;
+  border-top-left-radius: 0 !important;
+  border-bottom-left-radius: 0 !important;
+
+  // make the left red border square (v-cards are otherwise rounded)
+  &.v-card {
+    border-top-left-radius: 0 !important;
+    border-bottom-left-radius: 0 !important;
+  }
+}
```

Comment:
Can you use the base class `invalid-section` here instead of duplicating things?

Reply:
the invalid-section class includes turning the title-labels red, which i do not need in this case

Reply:
Are you doing anything with class "title-label"? If not, you could just remove it.

Reply:
oops thanks, will remove

Reply:
Can you remove this CSS now?

Reply:
the standard invalid-section class turns the labels red in addition to adding the red side bar but there are certain cases  where I do not need both

Reply:
I thought you got around that by removing the `title-label` class from your label?

Reply:
the special class is used in the documents delivery section as well

Reply:
OK, I see.

I would prefer to see an override for title-label in DocumentDelivery for this on-off special case instead of duplicating the base CSS here. (You can do this using `::v-deep`.)

Reply:
I see, I can only find examples of how to alter or add additional styling within v-deep online. How can I remove the title-label styling in the override entirely?

Reply:
There are 62 examples of using v-deep in Create UI.

In short, you need to "reach into" (using v-deep) the DocumentDelivery component and override the colour CSS property.

Reply:
thanks will switch it

Reply:
![Screen Shot 2022-07-11 at 2 06 12 PM](https://user-images.githubusercontent.com/73866619/178360482-e28270ed-4926-477d-9043-3112870f378c.png)

Reply:
![Screen Shot 2022-07-11 at 2 21 45 PM](https://user-images.githubusercontent.com/73866619/178360579-2bf1f70f-8c81-4ac7-aeb3-1d4191cf15d7.png)

Reply:
![Screen Shot 2022-07-11 at 2 23 03 PM](https://user-images.githubusercontent.com/73866619/178360784-4b171dc4-be9f-45bc-a113-55d23b994609.png)

Reply:
trying to override the title-label styling in the invalid-section within the document-delivery-section as you suggested, is my approach wrong

Reply:
I see 3 places that the code sets the "invalid-section" class on the DocumentDelivery component, so you may need this override in all places.

I think this CSS might work:
```
// override the title labels so they don't change colour on error
::v-deep #document-delivery .title-label {
  color: $gray7 !important;
}
```

Or maybe you will need to more precisely select the label (since CSS has an order of operation).

Reply:
// override the title labels so they don't change colour on error
::v-deep #document-delivery-section .invalid-section.title-label {
  color: $gray7 !important;
}

i tried what you suggested as well as the snippet of code above with more specific labels, neither appear to work, unsure if i am missing something else